src/main/java/org/spongepowered/api/plugin/Plugin.java

```diff
@@ -55,6 +55,15 @@
     String name();
 
     /**
+     * Gets the prefixes that can be prepended to a command to indicate this
+     * plugin's version of said command. This must have at least one item.
+     * 
+     * @return The prefix prepended to a command to indicate this plugin's
+     *         version of said command.
+     */
+    String[] commandPrefixes();
```

Comment:
@Deamon5550 Plugins may want to have multiple, minecraft almost certainly would. (mc:give, minecraft:give), I don't see why not to open this option up to plugins. Though I will change the fallback from .getName() to .getId()